utils/litemap/Cargo.toml

```diff
@@ -43,7 +44,7 @@ criterion = { workspace = true }
 default = ["alloc"]
 alloc = []
 databake = ["dep:databake"]
-serde = ["dep:serde"]
+serde = ["dep:serde", "alloc"]
```

Comment:
Note for reviewers: The addition of alloc was theoretically a breaking change for people using "no default features" so we may want to bump litemap to 0.8.